FunNet/Classes/Multipart/FormDataEncoder.swift

```diff
@@ -54,9 +58,11 @@ public struct FormDataEncoder {
 
 // MARK: Private
 private final class FormDataEncoderContext {
-  var parts: [MultipartComponent]
-  init() {
-    self.parts = []
+  var shouldConvertToCamelCase: Bool
+  var parts = [MultipartComponent]()
+  
+  init(_ shouldConvertToCamelCase: Bool) {
+    self.shouldConvertToCamelCase = shouldConvertToCamelCase
```

Comment:
If you're concerned about using ternary operators, we could instead store a function variable (String) -> String that we use to transform the keys, and then have a ternary operator in the init that sets the function to either `camelToSnake` or `id`?

Reply:
Great thought – file an issue and do that later!